taken from upstream tree

https://bugs.gentoo.org/417549

From dcc1615fd81c9a87677cd5b7138c12eab997b60f Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <bonzini@gnu.org>
Date: Fri, 20 Nov 2009 10:12:01 +0000
Subject: [PATCH] sync config with gcc:

2009-10-23  Rainer Orth  <ro@CeBiTec.Uni-Bielefeld.DE>

	* acx.m4 (ACX_CHECK_CYGWIN_CAT_WORKS): Use = with test.
---
 config/ChangeLog |    4 ++++
 config/acx.m4    |    2 +-
 2 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/config/acx.m4 b/config/acx.m4
index cea08b7..b559c03 100644
--- a/config/acx.m4
+++ b/config/acx.m4
@@ -605,7 +605,7 @@ dnl # See binutils PR 4334 for more details.
 AC_DEFUN([ACX_CHECK_CYGWIN_CAT_WORKS],[
 AC_MSG_CHECKING([to see if cat works as expected])
 echo a >cygwin-cat-check
-if test `cat cygwin-cat-check` == a ; then
+if test `cat cygwin-cat-check` = a ; then
   rm cygwin-cat-check
   AC_MSG_RESULT(yes)
 else
-- 
1.7.8.6

